### ISSUE TITLE
Doesn't work

### ISSUE THREAD

**Issue by narcolepticinsomniac** [MorbZ/HTML5-Video-Resumer#2]
Haven't found one that works well globally. Had high hopes for this one. I assume it once worked, but not currently in Chromium 62.0.3202.75.

**Comment by Croydon** [comment]
I have tested this project as well and it basically didn't work for me at all.
This one works at least for YouTube: https://addons.mozilla.org/en-US/firefox/addon/video-resumer

Would be nice to have an extension for all HTML5 videos.

**Comment by narcolepticinsomniac** [comment]
Yeah, I'm interested in a global solution, which this one supposedly is. It also uses a Mutation Observer, which would be ideal for dynamically added players, but unfortunately it's broken. I've had no luck trying to fix it myself, but maybe we'll get lucky and @MorbZ will update it.

**Comment by MorbZ** [comment]
The problem is that Youtube and some other sites use "blob:http://" urls for their videos. Those URLs change every time you visit the page or change the quality. Currently the extension only supports videos with static URLs. But I'll see what I can do.

**Comment by narcolepticinsomniac** [comment]
I found one that works with pretty much any video, but it's nowhere near as sophisticated as yours aims to be. The major flaws are that it only saves the place in the timeline for the last video watched, and there's no mutation observer, so it only works for videos which load along with the page. Therefore if you test with Youtube, you probably need to open vids in a new tab since they load via pushState (I believe).

IDK enough to know what IDK, but perhaps there's something that could be incorporated as far as saving timeline progress for non-static URLs. 

https://github.com/damre/HTML5VideoTimeRecorder

I added `<all_urls>` and `"all_frames": true` so it works everywhere including iframes.

[HTML5VideoTimeRecorder-master.zip](https://github.com/MorbZ/HTML5-Video-Resumer/files/1476996/HTML5VideoTimeRecorder-master.zip)

**Comment by thetiby** [comment]
Hey guys, I'm dropping this here in case it's useful to anyone: https://github.com/thetiby/HTML5-Video-Resumer/commit/bd2e492f6e50cf61b2d066605653e40c229aeec8

I did a fork on this and added another file which can be used without the extension itself (but it does require a different extension to inject the code. The file is only there for later reference.
